Fix typos (`the the` > `the`)

eliasp opened the pull request:
It looks like humans aren't only very bad at recognizing a double `the` in a sentence when reading [1], but they're also very good at sneaking them in unintentionally without noticing.

Fix this by running:
```
ack --ignore-dir=doc/man --ignore-dir=doc/topics/releases -i 'the the\W' -l | xargs sed -i 's|the the\(\W\)|the\1|g'
```

[1] Eye movements and word skipping during reading: Effects of word length and predictability: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3543826/

### What does this PR do?
Fixes a common typo

### What issues does this PR fix or reference?
None

### Tests written?
No

### Commits signed with GPG?
Yes

waynew commented:
Hey @eliasp looks like we missed merging this a while ago. Do you mind rebasing this on `master`?

Might be worth just running this against master to see if there are *more* double `the`'s there :trollface:

eliasp committed:
Fix typos (`the the` > `the`)

It looks like humans aren't only very bad at recognizing a double `the` in a sentence when reading [1], but they're also very good at sneaking them in unintentionally without noticing.

Fix this by running:
```
ack --ignore-dir=doc/man --ignore-dir=doc/topics/releases -i 'the the\W' -l | xargs sed -i 's|the the\(\W\)|the\1|g'
```

[1] Eye movements and word skipping during reading: Effects of word length and predictability: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3543826/

eliasp commented:
Rebased on `master` + re-created the commit by running it again.

eliasp commented:
Failing CI is completely unrelated (and only editing doc/comments shouldn't have any influence on CI anyways 😄)

eliasp commented:
Rebased against latest master to get some merge conflicts ouf of the way

Ch3LL commented:
looks like there are some failure that need to be resolved here. Looks like the rebase might have caused some issues. We recently added pre-commit which should run black against your code. I noticed that the PR has some single ticks where black requires double quotes.

dwoz committed:
Blacken changed files

dwoz committed:
Blacken changed files

waynew committed:
Merge remote-tracking branch 'salt/master' into HEAD

s0undt3ch commented on doc/topics/development/tests/unit.rst:
```diff
@@ -174,8 +174,13 @@ What happens in the above example is we mock a call to
+import patching
+import running.
+import tests
+import when
```
All these imports.... :grin:

waynew replied:
:eyes: 

:fire:

s0undt3ch commented on doc/topics/tutorials/firewall.rst:
```diff
@@ -207,7 +207,7 @@ Whitelist communication to Master
 =================================
 
 There are situations where you want to selectively allow Minion traffic
-from specific hosts or networks into your Salt Master. The first
```
Unintentional delete?

waynew replied:
fixed

waynew committed:
Re-add unintentional delete

waynew committed:
Bad merge fix

waynew committed:
Fix broken docstring

waynew committed:
Merge salt/master

waynew committed:
Another try

waynew committed:
pre-commit should pass now?

waynew reviewed (approved):
🎉

waynew commented:
@s0undt3ch are you good here?
